knative-operator/deploy/resources/console_cli_download_kn_resources.yaml

```diff
@@ -1,77 +0,0 @@
-apiVersion: apps/v1
```

Comment:
I just noticed (sorry for the late comment): What happens to the old things on update? I don't think we're removing them anywhere, are we?